### ISSUE TITLE
Panic when trying to use Uptrace with fluent-bit

### ISSUE THREAD

**Issue by jeffbicca** [uptrace/uptrace#214]
Before setting up a fluent-bit, the environment I had with Uptrace previously was running fine (obviously without getting logs into Uptrace).

But after trying to up an instance of fluent-bit and configure it like in [this configuration](https://uptrace.dev/get/ingest/fluent-bit.html#configuration), just right when the first log message hits in fluent-bit, I get the following stack trace from Uptrace:

```
panic: runtime error: invalid memory address or nil pointer dereference

goroutine 181 [running]:
github.com/uptrace/uptrace/pkg/httputil.PanicHandler.ServeHTTP.func1()
        github.com/uptrace/uptrace/pkg/httputil/middleware.go:24 +0x65
panic({0x112a5e0, 0x312d1c0})
        runtime/panic.go:884 +0x213
go.opentelemetry.io/otel/sdk/trace.(*recordingSpan).End.func1()
        go.opentelemetry.io/otel/sdk@v1.14.0/trace/span.go:383 +0x2a
go.opentelemetry.io/otel/sdk/trace.(*recordingSpan).End(0xc0015b4000, {0x0, 0x0, 0x4ca13b?})
        go.opentelemetry.io/otel/sdk@v1.14.0/trace/span.go:421 +0xa89
panic({0x112a5e0, 0x312d1c0})
        runtime/panic.go:884 +0x213
github.com/uptrace/uptrace/pkg/tracing.(*LogsServiceServer).export(0xc0005a6760, {0x290e820, 0xc0015842d0}, {0xc001594230, 0x1, 0x4?}, 0xc001586180)
        github.com/uptrace/uptrace/pkg/tracing/otlp_logs.go:152 +0xa7
github.com/uptrace/uptrace/pkg/tracing.(*LogsServiceServer).ExportHTTP(0xc0005a6760, {0x290e470, 0xc000666200}, {0xc00159c300, {{0xc00159801d, 0x8}, 0xc000361200, 0xc0005a6860, 0x0}})
        github.com/uptrace/uptrace/pkg/tracing/otlp_logs.go:98 +0x3f0
github.com/uptrace/bunrouter/extra/bunrouterotel.(*config).Middleware.func1({0x290e470, 0xc000666200}, {0xc00159c300, {{0xc00159801d, 0x8}, 0xc000361200, 0xc0005a6860, 0x0}})
        github.com/uptrace/bunrouter/extra/bunrouterotel@v1.0.20/bunrouterotel.go:70 +0xb36
github.com/uptrace/uptrace/pkg/bunapp.(*App).httpErrorHandler.func1({0x290e470, 0xc000666200}, {0xc00159c300, {{0xc00159801d, 0x8}, 0xc000361200, 0xc0005a6860, 0x0}})
        github.com/uptrace/uptrace/pkg/bunapp/app.go:273 +0x88
github.com/uptrace/bunrouter/extra/reqlog.(*middleware).Middleware.func1({0x290e650?, 0xc0006661c0?}, {0xc00159c300, {{0xc00159801d, 0x8}, 0xc000361200, 0xc0005a6860, 0x0}})
        github.com/uptrace/bunrouter/extra/reqlog@v1.0.20/middleware.go:78 +0x123
github.com/uptrace/bunrouter.(*Router).ServeHTTPError(0xc0006661c0?, {0x290e650, 0xc0006661c0}, 0xc00159c300)
        github.com/uptrace/bunrouter@v1.0.20/router.go:52 +0x128
github.com/uptrace/bunrouter.(*Router).ServeHTTP(0x1239380?, {0x290e650?, 0xc0006661c0?}, 0x4?)
        github.com/uptrace/bunrouter@v1.0.20/router.go:45 +0x25
github.com/klauspost/compress/gzhttp.NewWrapper.func1.1({0x290e3b0, 0xc000660240}, 0xc0015ac000?)
        github.com/klauspost/compress@v1.16.5/gzhttp/compress.go:484 +0x4f2
net/http.HandlerFunc.ServeHTTP(0x11dfac0?, {0x290e3b0?, 0xc000660240?}, 0xe?)
        net/http/server.go:2122 +0x2f
github.com/uptrace/uptrace/pkg/httputil.DecompressHandler.ServeHTTP({{0x29076a0?, 0xc0001c32c0?}}, {0x290e3b0, 0xc000660240}, 0xc00159c300)
        github.com/uptrace/uptrace/pkg/httputil/middleware.go:52 +0x1b6
github.com/uptrace/uptrace/pkg/httputil.(*TraceparentHandler).ServeHTTP(0xc0001c32e0, {0x290e3b0, 0xc000660240}, 0xc00159c300)
        github.com/uptrace/uptrace/pkg/httputil/middleware.go:89 +0xb9
go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp.(*Handler).ServeHTTP(0xc000001bc0, {0x290d660?, 0xc0015ae000}, 0xc00159c100)
        go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp@v0.40.0/handler.go:213 +0x1231
github.com/rs/cors.(*Cors).Handler.func1({0x290d660, 0xc0015ae000}, 0xc00159c100)
        github.com/rs/cors@v1.9.0/cors.go:236 +0x1c4
net/http.HandlerFunc.ServeHTTP(0x0?, {0x290d660?, 0xc0015ae000?}, 0x1940f133a?)
        net/http/server.go:2122 +0x2f
github.com/uptrace/uptrace/pkg/httputil.PanicHandler.ServeHTTP({{0x29076a0?, 0xc0001c33a0?}}, {0x290d660?, 0xc0015ae000?}, 0xc0015a5ad0?)
        github.com/uptrace/uptrace/pkg/httputil/middleware.go:30 +0x5c
net/http.serverHandler.ServeHTTP({0x290a508?}, {0x290d660, 0xc0015ae000}, 0xc00159c100)
        net/http/server.go:2936 +0x316
net/http.(*conn).serve(0xc001596000, {0x290e820, 0xc0014d3b90})
        net/http/server.go:1995 +0x612
created by net/http.(*Server).Serve
        net/http/server.go:3089 +0x5ed
```

Some additional information about the versions I am using:
- Uptrace v. 1.4.3
- OpenTelemetry Collector v. 0.7.5.0
- fluent-bit v. 2.0.9

**Comment by vmihailenco** [comment]
Thanks for reporting! This should be fixed in v1.4.5.

**Comment by jeffbicca** [comment]
Already tested it and it's working flawlessly now.
Thank you!
